ignite/contrib/handlers/tqdm_logger.py

```diff
@@ -145,7 +145,13 @@ def log_message(self, message):
             message (str): string you wish to log.
         """
         from tqdm import tqdm
-        tqdm.write(message, **self.tqdm_kwargs)
+
+        if "file" in self.tqdm_kwargs:
```

Comment:
can we replace this part with `tqdm.write(message, file=self.tqdm_kwargs.get('file', None))`?

Reply:
Sure! That's way more elegant.